src/hooks/words/use-word-by-id.hook.ts

```diff
@@ -6,7 +6,7 @@ import { useApiErrorHook } from '../use-api-error.hook'
 export const useWordById = (id: string) => {
   const handleApiError = useApiErrorHook()
 
-  const handleRefresh = useRecoilCallback(
+  const onWordById = useRecoilCallback(
```

Comment:
onGetWordById

Reply:
Handled in [5ddf33e](https://github.com/ajktown/wordnote/pull/158/commits/5ddf33e05136c5e01ce02ca05840754462e6215c)